src/CMakeLists.txt

```diff
@@ -678,9 +683,9 @@ add_executable(ceph-osd ${ceph_osd_srcs}
   $<TARGET_OBJECTS:common_util_obj>)
 add_dependencies(ceph-osd erasure_code_plugins)
 target_link_libraries(ceph-osd osd os global ${BLKID_LIBRARIES})
-if(WITH_FUSE)
+if(HAVE_LIBFUSE)
```

Comment:
i don't think we want this change. one can always disable fuse using `-DWITH_FUSE=OFF`.